docs/HowTo/Interact/APIs/API.md

```diff
@@ -52,10 +52,16 @@ Ports must be [exposed appropriately](../../Find-and-Connect/Managing-Peers.md#p
 
 ## Host allowlist
 
-To prevent DNS rebinding, Besu accepts incoming HTTP requests, WebSockets connections, and GraphQL
-requests only from hostnames specified using the
-[`--host-allowlist`](../../../Reference/CLI/CLI-Syntax.md#host-allowlist) option. Besu accepts
-incoming requests and connections from `localhost` and `127.0.0.1` by default.
+To prevent DNS rebinding attacks, Besu checks incoming HTTP request host headers, WebSockets connections, and GraphQL
+requests.
+The API accepts requests only when hostnames specified using the
+[`--host-allowlist`](../../../Reference/CLI/CLI-Syntax.md#host-allowlist) option matches the request host headers.
+Besu accepts incoming requests and connections to `localhost` and `127.0.0.1` by default.
```

Comment:
Not sure if I'm reading into it too much, but is it the API or Besu accepting a request? Should it be "Besu accepts requests..." or "The API processes requests..." instead?